libs/payments/ui/src/lib/client/components/Breadcrumbs/index.tsx

```diff
@@ -62,8 +62,41 @@ export function Breadcrumbs(args: {
   }
 
   return (
-    <nav className="p-4 tablet:p-6" aria-label="Breadcrumb">
-      <ol className="flex items-center">
+    <nav
+      className="
+        p-4 tablet:p-6
+        border-b tablet:border-b-0
+        [border-bottom-color:var(--border-stroke-color-divider,#E0E0E6)]
+      "
+      aria-label="Breadcrumb"
+    >
+      <ol className="flex items-center tablet:hidden">
+        {breadcrumbs.map(({ label, href }, i) => {
+          const isPrev = i === breadcrumbs.length - 2;
+          if (!isPrev) return null;
+
+          const url = new URL(href);
+          url.search = searchParams.toString();
+          href = url.href;
+
+          return (
+            <li className="flex items-center" key={`breadcrumb-nav-${i}`}>
+              <Link
+                href={href}
+                className="inline-flex items-center font-semibold text-blue-600"
```

Comment:
We can add `gap-3` (gap: 12px, that's the closest to 10px) and remove the inline styles below now.

Reply:
To improve accessibility, we should add more context to this button since the chevron arrow is not descriptive on its own. 

Adding an `aria-label` lets assistive technology (e.g screen readers) provide a more meaningful description, like "Go back to (page)", which will improve navigation (see [example](https://github.com/mozilla/fxa/blob/main/apps/payments/next/app/%5Blocale%5D/subscriptions/manage/page.tsx#L366-L372)).